### DISCUSSION TITLE
[Tip] No sound in applications/games fix

### DISCUSSION THREAD

**Discussion by Cthululz** [sandboxie-plus/Sandboxie#4441] [Documents and notes]
tl;dr: Deleting `C:\windows\system32\*` in the affected sandbox resolved the issue for all my games

I recently had an issue where virtually all of my games had no sound any longer when run in a sandbox that I have been using for over a year.  There were maybe 2 games out of about 50 that had sound but those were extremely old games not using newer DirectX for anything.

I tried reinstalling certain games that had been working fine just a week ago but the problem persisted.  This was maddening.

I backed up my whole sandbox to an external drive in case I had to start all over or borked something while trying to resolve this issue.  I couldn't really find anything online for the same problem, save for a few people posting about the same issue, but never posting any resolutions.

I created a new sandbox with the same settings as the previous and copied one of the game folders over to it.  I fired up the game and sound worked!  That helped me narrow down the possibilities, but copying all of my games this way would not work as some of them were relying on files in other folders and registry keys and the like.  Copying everything from one sandbox to the other brought the same problem of no sound again.

I ended up find some DirectX install error logs while digging through various folders.  I ran dxdiag from the problematic sandbox and it gave errors about not being able to load extra sound device information.  When I ran dxdiag from my non-sandboxed machine I had no such errors, and I also had no errors when I ran dxdiag from a brand new sandbox with nothing installed.

I tried various things to try to reinstall DirectX and drivers and other things but nothing worked.

Making sure I had everything backed up, I ended up deleting everything within the broken sandbox's `C:\windows\system32 ` folder.  Directly after having done so, all my games are now working with sound again.  I think something had tried to install an older DirectX version and it failed for some reason and that screwed up all the sound for the rest of my games.  

Weirdly they still ran fine but just had no sound nor any other indication that anything was wrong. 

I just wanted to post this somewhere in case someone else runs into the same issue later on and this may save them many hours of headaches.
